src/app/services/dirty-dialog.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+
+/**
+ * Async modal dialog service
+ * DialogService makes this app easier to test by faking this service.
```

Comment:
Do you mean "DirtyDialogService"?
@the-greenman

Reply:
Was an old comment anyway. Removed